docs/deploy-development-rp.md

```diff
@@ -145,7 +145,8 @@
 
 ### Metrics
 
-To run fake metrics socket:
+You can watch the output of the cluster metrics:
+
 ```bash
-nc -U -l mdm_statsd.socket
+go run ./hack/monitor
```

Comment:
I think, this PR will be a big bigger than you initially thought. Could you please move this change & script into a own PR so the team can review & discuss it separately from the feature.

Reply:
I've split this out into [PR 333](https://github.com/Azure/ARO-RP/pull/333)